app/src/main/java/org/oppia/app/options/OptionsFragment.kt

```diff
@@ -0,0 +1,197 @@
+package org.oppia.app.options
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.Observer
+import androidx.lifecycle.Transformations
+import androidx.preference.Preference
+import androidx.preference.PreferenceFragmentCompat
+import androidx.preference.PreferenceManager
+import org.oppia.app.R
+import org.oppia.app.home.KEY_HOME_PROFILE_ID
+import org.oppia.app.model.Profile
+import org.oppia.app.model.ProfileId
+import org.oppia.domain.profile.ProfileManagementController
+import org.oppia.util.data.AsyncResult
+import org.oppia.util.logging.Logger
+import javax.inject.Inject
+
+const val STORY_TEXT_SIZE_SMALL = "Small"
+const val STORY_TEXT_SIZE_MEDIUM = "Medium"
+const val STORY_TEXT_SIZE_LARGE = "Large"
+const val STORY_TEXT_SIZE_EXTRA_LARGE = "Extra Large"
+
+const val MESSAGE_STORY_TEXT_SIZE = "Text Size"
+const val MESSAGE_APP_LANGUAGE = "App Language"
+const val MESSAGE_AUDIO_LANGUAGE = "Audio Language"
+
+class OptionsFragment @Inject constructor(
```

Comment:
`Options + Cmd + L` to fix the formatting issues in this file.

Reply:
Done.